quicksort/quick_sort.py

```diff
@@ -1,3 +1,18 @@
+def median(a, i, j, k):
+    """
+    Returns Median of 3 integers from array a.
+    :param a: Iterable of elements
+    :param i: index of element of set, whose meadian is to be returned
+    :param i: index of element of set, whose meadian is to be returned
+    :param i: index element of set, whose meadian is to be returned
+    :return: returns the vale of median of a[i], a[j], a[k].
```

Comment:
Misleading information here, this function here returns the index of median not the actual value.